Scripts/Service/Unity/UnityHttpRequest.cs

```diff
@@ -125,8 +125,7 @@ private void UpdateProgress(ref float currentProgress, float progress, Action<fl
 		{
 			if (currentProgress < progress)
 			{
-				currentProgress = progress;
-				onProgress(downloadProgress);
+				onProgress?.Invoke(currentProgress = progress);
```

Comment:
With the assigning of currentProgress happening in the invocation does this mean it wont be assigned if onProgress event is null? if so then this would be incorrect functionality.